static-site/content/blog/2024-08-30-augur-v25.md

```diff
@@ -0,0 +1,122 @@
+---
+author: "The Nextstrain team"
+date: "2024-08-30"
+title: "Notable changes in Augur v25"
+---
+
+The Nextstrain team has released several new versions of Augur – our
+bioinformatics toolkit that assists with phylogenomic analyses – over
+the past few months. We wanted to highlight some of the significant
+feature improvements in these new releases.
+
+### Excel and OpenOffice support
+
+One of the features we are most excited to announce is that the
+[`augur curate`][] command now supports both Excel (`.xlsx` and
+`.xls`) and OpenOffice (`.ods`) files as metadata inputs. This allows
+the easy conversion of Excel or OpenOffice files into the expected
+metadata TSV used by other Augur commands. *This new feature is
+available in Augur v25.2 and later.*
```

Comment:
These availability statements are great, but it makes me realize we should be using

```rst
.. versionadded:: x.y.z
```

directives in our Sphinx/rST docs so those availability statements are also there.

Reply:
Created [tracking issue in the augur repo](https://github.com/nextstrain/augur/issues/1612) for this, will maybe try to do that in the next 2-week sprint cycle...